.github/workflows/compliance.yml

```diff
@@ -1,15 +1,20 @@
 name: Compliance Checks
 
 on:
-  pull_request:
-    types:
-    - edited
-    - opened
-    - reopened
-    - synchronize
+  workflow_call:
 
 jobs:
+  check_compliance_push:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-22.04
```

Comment:
is it time we move all gh actions to 24.04?

Reply:
yeah, need to do this after that...